apps/site/pages/en/learn/modules/package-configuration.md

```diff
@@ -0,0 +1,406 @@
+---
+title: Package Configuration
+layout: learn
+authors: JakobJingleheimer
+---
+
+# Package configuration
+
+Configuration is always a chore, but an unfortunately necessary evil. And configuring a package for CommonJS (CJS) and ES Modules (ESM) can be a waking nightmare—not least because it has changed a dozen times in half as many years.
+
+A frequent question is “how do I make this work!?” (often with angry tears); but yet more frequently we come across packages that are just misconfigured.
+
+All the provided `package.json` configurations (not specifically marked “does not work”) work in Node.js 12.22.x (v12 latest, the oldest supported line) and 17.2.0 (current latest at the time)<sup>[1](#footnotes)</sup>, and for grins, with webpack 5.53.0 and 5.63.0 respectively. These are available: [nodejs/package-examples](https://github.com/nodejs/package-examples/blob/main/config).
```

Comment:
The markdown parser supports [footnotes](https://www.markdownguide.org/extended-syntax/#footnotes), IMO they should be used here, instead of the `<sup>...</sup>`

(E.G):
```
Some text[^1]

[^1]: Additional content
```

Some text[^1]

[^1]: Additional content